README.md

```diff
@@ -128,33 +98,42 @@ Make sure to use the settings shown below.
 
 | Parameter | Description |
 |---|---|
-| Resource group | Click **Create new**. Give a meaningful **Name**, such as "logziobBlobStorageIntegration", and then click **OK**. |
+| Resource group | Select your existing resource group, and then click **OK**. |
 | Location | Select the same region as the Azure services that will stream data to this Blob Storage. |
-
-#### In the SETTINGS section
-
-| Parameter | Description |
-|---|---|
 | Logs listener host | Use the listener URL specific to the region of your Logz.io account. You can look it up [here](https://docs.logz.io/user-guide/accounts/account-region.html). |
 | Logs account token | Add the [log shipping token](https://app.logz.io/#/dashboard/settings/general) for the relevant Logz.io account. This is the account you want to ship to.  |
-| Format (Default: text) | Select one of the supported parsing formats: json/csv/text  |
+| Format (Default: text) | Select one of the supported parsing formats: text/json/csv  |
+| Buffersize (Default: 100) | The maximum number of messages the logger will accumulate before sending them all as a bulk  |
+| Timeout (Default: 180,000 = 3 minutes) | The read/write/connection timeout in *milliseconds*.  |
 
-At the bottom of the page, agree to the terms and conditions.
-Then click **Purchase** to deploy. Deployment can take a few minutes.
+At the bottom of the page, agree to the terms and conditions. 
+Then click **Purchase** to deploy. 
 
 Deployment can take a few minutes.
 
-### 2. Check Logz.io for your logs
+## Check Logz.io for your logs
 
 Give your logs some time to get from your system to ours, and then open [Kibana](https://app.logz.io/#/dashboard/kibana/discover?). 
-Any logs sent from this point on should appear under the type `blobStorage`.
+Please note that only logs sent from this point on would be sent and appear under the type `blobStorage`.
 
 If you still don’t see your logs, see [log shipping troubleshooting](https://docs.logz.io/user-guide/log-shipping/log-shipping-troubleshooting.html).
 
 </div>
 
 </div>
 <!-- tab:end -->
+<div id="parameters-after-deploy">
+
+### Working with your parameters after deployment
+
+If you wish to change parameters values after the deployment, go to your function app page, then on the left menu press the 'Configuration' tab.
+You'll have the option to edit the following values:
+* Shipper's configurations such as LogzioHost, LogzioToken, Buffersize, Timeout.
+* FUNCTIONS_WORKER_PROCESS_COUNT - maximum of 10, for more information press [here](https://docs.microsoft.com/en-us/azure/azure-functions/functions-app-settings#functions_worker_process_count).
+* ParseEmptyField - Parse logs with invalid empty fields. **Please note using this option may slow the shipper's perfomance.**
```

Comment:
```suggestion
### Updating parameters after deployment

Some parameters can be updated post-deployment. These include:

* Shipper-related configurations: **LogzioHost**, **LogzioToken**, **Buffersize**, **Timeout**.
* **FUNCTIONS_WORKER_PROCESS_COUNT** - maximum of 10. [See Microsoft documentation for more details](https://docs.microsoft.com/en-us/azure/azure-functions/functions-app-settings#functions_worker_process_count).
* **ParseEmptyField** - Enable/disable the option to parse logs with invalid empty fields. This option exists to overcome a rare bug whereby the services ship unnamed fields that break the parsing pipeline. **Please consider enabling this option only if you encounter unparsed logs due to unnamed fields. Note that it may slow the shipper's performance.**

To update your parameters post-deployment, open the **Function App** page in your Azure portal. On the left menu, select the **Configuration** tab and edit the relevant values.

```